### PR TITLE
ore: implement 'gc' subcommand for aws

### PR DESCRIPTION
**Pull request by euank**
The intent is that running this in a distributed-cron-like-thing should be enough to generally catch leaks.

### CODE REVIEW

File: cmd/ore/aws/gc.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	"time"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	cmdGC = &cobra.Command{
+		Use:   "gc",
+		Short: "GC resources in AWS",
+		Long:  `Delete GC instances created over the given duration ago`,
```
Comment (bgilbert) [inline cmd/ore/aws/gc.go]: `s/GC //`

File: cmd/ore/aws/gc.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+		RunE:  runGC,
+	}
+
+	gcDuration time.Duration
+)
+
+func init() {
+	AWS.AddCommand(cmdGC)
+	cmdGC.Flags().DurationVar(&gcDuration, "duration", 2*time.Hour, "how old resources must be before they're considered garbage")
```
Comment (bgilbert) [inline cmd/ore/aws/gc.go]: Defaulting to two hours seems a bit aggressive.  6?  24?

Comment (euank) [reply cmd/ore/aws/gc.go]: I split the difference and picked 5

**Comment by euank** [comment]
@arithx They won't for `kola spawn` in the general case, only in the case where you use an account that's cronning `ore gc`... that's more a "this aws account does this" documentation thing I guess?

**Comment by bgilbert** [comment]
Also, that really only applies to folks using the team AWS account.

**Pull request merged**